src/docs/product/issues/performance-issues/n-one-queries.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: "N+1 Queries"
+sidebar_order: 10
+description: "Learn more about N+1 queries and how to diagnose and fix them."
+---
+
+<Include name="early-adopter-note.mdx" />
+
+## N+1 Queries
+
+"N+1 Queries" is a performance problem where the application makes database queries in a loop, instead of making a single query that returns all the information at once. Each database connection has some amount of performance overhead, so querying the database in a loop can be many times slower than doing it just once. This problem often occurs when using an Object-relational mapping (ORM) tool in web frameworks like Django or Ruby on Rails.
+
+## Span Evidence
+
+The evidence for a N+1 queries has four main aspects:
```

Comment:
Are you talking about in general or in the context of the performance issue detail? If the latter, then change this to "N+1 query issue"

```suggestion
The evidence for an N+1 query has four main aspects:
```

Reply:
Changed to "The evidence for an N+1 queries problem has..." for consistency!